### PR TITLE
Start of customize server

### PR DESCRIPTION
**Pull request by mbfrahry**
Signed-off-by: Matthew Frahry <matthew.frahry@hpe.com>

### CODE REVIEW

File: ov/connectionv3.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// Package ov -
+package ov
+
+import (
+	"github.com/HewlettPackard/oneview-golang/utils"
+)
+
+func (c *OVClient) ManageI3SConnections(connections []Connection) []Connection {
+
+	if len(connections) < 4 {
```
Comment (wenlock) [inline ov/connectionv3.go]: should this be <= ??

File: ov/connectionv3.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		connection2 := Connection{
+			ID:            2,
+			Name:          "Deployment Network B",
+			FunctionType:  "Ethernet",
+			RequestedMbps: "2500",
+			NetworkURI:    utils.NewNstring("/rest/ethernet-networks/8d4a7e7f-2c9d-4d79-be86-defd0dd5e8cf"),
+			Boot:          boot2,
+			PortID:        "Mezz 3:2-a",
+		}
+		connections = append(connections, connection1)
```
Comment (wenlock) [inline ov/connectionv3.go]: Is connections sorted on ID?  what happens if there is already a connection with ID = 1 or 2?

File: ov/connectionv3.go (hunk 1 above)
Comment (mbfrahry) [inline ov/connectionv3.go]: It is correct the way it is. There are 2 connections for our profile template. And we want to add 2 more...unless they are already there. I don't like it but this is temporary until we can figure out what we want to do

File: ov/connectionv3.go (hunk 2 above)
Comment (mbfrahry) [inline ov/connectionv3.go]: The connections for our template are 3 and 4 so I hard-coded 1 and 2. It won't work if 1 and 2 are taken.

**Comment by mbfrahry** [comment]
I didn't add any logic into the connection piece  because I don't know what the overall plan is. Let's hold off on the merge until we talk to bob

**Pull request closed without merging**